tests/components/orderentry/Accordion.test.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Accordion from '../../../app/js/components/orderEntry/Accordion';
+
+let props;
+let mountedComponent;
+
+const getComponent = () => {
+  if (!mountedComponent) {
+    mountedComponent = shallow(<Accordion {...props} />);
+  }
+  return mountedComponent;
+};
+
+describe('Component: Accordion', () => {
+  beforeEach(() => {
+    props = {
+      open: true,
+      title: 'Header',
+      children: [
+        <h1>I love this</h1>
+      ]
+    };
+    mountedComponent = undefined;
+  });
+
+  it('renders properly', () => {
+    const component = getComponent();
+    expect(component).toMatchSnapshot();
+  });
+
+  it('toggles the content visibility when clicked', () => {
+    const renderedComponent = getComponent();
+    const toggleViewLink = renderedComponent.find('.accordion-link')
+    toggleViewLink.simulate('click');
+    expect(renderedComponent.state('isVisible')).toEqual(false);
```

Comment:
Assuming `isVisible` was false before. This would still pass. How is this:
Expect `isVisible` to be `true` then `click` and expect the `reverse`. What do you think?

Reply:
This accordion component, when it receives a prop of open to be true by defaults sets the isVisible state to true, so when clicked this time around is set to false. you can see that the open prop was parsed to this accordion component in line 17.